services/models/workload.go

```diff
@@ -0,0 +1,39 @@
+package models
+
+import "k8s.io/api/apps/v1beta1"
+
+type WorkloadList struct {
+	// Namespace where the workloads live in
+	// required: true
+	// example: bookinfo
+	Namespace Namespace `json:"namespace"`
+
+	// Workloads for a given namespace
+	// required: true
+	Workloads []WorkloadOverview `json:"workloads"`
+}
+
+type WorkloadOverview struct {
+	// Name of the workload
+	// required: true
+	// example: reviews-v1
+	Name string `json:"name"`
```

Comment:
This is merged, but I think the workload overview could have a field to identify which is the origin type.
i.e. Workload.Type == 'Deployment', Workload.Type == 'ReplicaSet', etc.
With that, in the future that field can be used for filtering.

Reply:
+1